pkg/statistics/handle/storage/read.go

```diff
@@ -655,18 +655,37 @@ func CleanFakeItemsForShowHistInFlights(statsCache statstypes.StatsCache) int {
 func loadNeededColumnHistograms(sctx sessionctx.Context, statsHandle statstypes.StatsHandle, col model.TableItemID, loadFMSketch bool, fullLoad bool) (err error) {
 	statsTbl, ok := statsHandle.Get(col.TableID)
 	if !ok {
+		// This could happen when the table is dropped after the async load is triggered.
+		statslogutil.StatsSampleLogger().Info(
+			"Table statistics item not found, possibly due to table being dropped",
```

Comment:
Some log messages are capitalized and some are not. Can we make them uniform?

Reply:
Updated.